.github/workflows/github_issues.yml

```diff
@@ -0,0 +1,22 @@
+name: Sync issue to Azure DevOps work item
+
+"on":
+  issues:
+    types:
+      [opened, edited, deleted, closed, reopened, labeled, unlabeled]
+
+jobs:
+  alert:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: danhellem/github-actions-issue-to-work-item@master
+        env:
+          ado_token: "${{ secrets.ADO_PERSONAL_ACCESS_TOKEN }}"
```

Comment:
is this needed ? would it impact ado setup?

Reply:
For this file existing on the preview branch, I don't foresee any conflicts with the version that we have on master. For the PAT in particular, we would have to regenerate it again post tomorrow's changes.